packages/bunyan/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import { Axiom } from '@axiomhq/js';
+
+export enum AxiomEventLevel {
+    Trace = 'trace',
+    Debug = 'debug',
+    Info = 'info',
+    Warn = 'warn',
+    Error = 'error',
+    Fatal = 'fatal',
+    Silent = 'silent',
+}
+
+
+enum RecordLevel {
+    trace = 10,
+    debug = 20,
+    info = 30,
+    warn = 40,
+    error = 50,
+    fatal = 60,
+  }
+const noop = () => {};
+export class AxiomStream <T extends {msg: string, level: RecordLevel, time: Date }>  {
+    private axiom: Axiom;
+    dataset: string;
+    token: string;
+    orgId: string;
+    onError: (err: unknown) => void 
+
+    constructor(opt: Omit<AxiomStream<T>, 'write' |'onError'> & {
+        onError?: AxiomStream<T>['onError'];
+    }){
+        if (!opt.dataset){
+            throw new Error('dataset requried')
+        }
+
+        if (!opt.orgId){
```

Comment:
hey, orgId is optional, users can use an API token. the orgId is required when users use a Personal Token.